sdk/src/main/java/com/silanis/esl/sdk/internal/converter/ReferencedFieldConditionsConverter.java

```diff
@@ -0,0 +1,45 @@
+        List<com.silanis.esl.api.model.FieldCondition> apiFieldConditionsInCondition = new ArrayList<com.silanis.esl.api.model.FieldCondition>();
+        for (com.silanis.esl.sdk.FieldCondition sdkFieldCondition : sdkReferencedFieldConditions.getReferencedInCondition()) {
+            FieldConditionConverter fieldConditionConverter = new FieldConditionConverter(sdkFieldCondition);
+            apiFieldConditionsInCondition.add(fieldConditionConverter.toAPIFieldCondition());
+        }
+
+        List<com.silanis.esl.api.model.FieldCondition> apiFieldConditionsInAction = new ArrayList<com.silanis.esl.api.model.FieldCondition>();
+        for (com.silanis.esl.sdk.FieldCondition sdkFieldCondition : sdkReferencedFieldConditions.getReferencedInAction()) {
+            FieldConditionConverter fieldConditionConverter = new FieldConditionConverter(sdkFieldCondition);
+            apiFieldConditionsInAction.add(fieldConditionConverter.toAPIFieldCondition());
+        }
+
+        com.silanis.esl.api.model.ReferencedFieldConditions apiReferencedFieldConditions = new com.silanis.esl.api.model.ReferencedFieldConditions();
+        apiReferencedFieldConditions.setReferencedInCondition(apiFieldConditionsInCondition);
+        apiReferencedFieldConditions.setReferencedInAction(apiFieldConditionsInAction);
+        return apiReferencedFieldConditions;
+    }
+
+    public static com.silanis.esl.sdk.ReferencedFieldConditions toSDK(com.silanis.esl.api.model.ReferencedFieldConditions apiReferencedFieldConditions) {
+        List<com.silanis.esl.sdk.FieldCondition> apiFieldConditionsInCondition = new ArrayList<com.silanis.esl.sdk.FieldCondition>();
+        for (com.silanis.esl.api.model.FieldCondition sdkFieldCondition : apiReferencedFieldConditions.getReferencedInCondition()) {
+            FieldConditionConverter fieldConditionConverter = new FieldConditionConverter(sdkFieldCondition);
+            apiFieldConditionsInCondition.add(fieldConditionConverter.toSDKFieldCondition());
+        }
+
+        List<com.silanis.esl.sdk.FieldCondition> apiFieldConditionsInAction = new ArrayList<com.silanis.esl.sdk.FieldCondition>();
+        for (com.silanis.esl.api.model.FieldCondition sdkFieldCondition : apiReferencedFieldConditions.getReferencedInAction()) {
+            FieldConditionConverter fieldConditionConverter = new FieldConditionConverter(sdkFieldCondition);
+            apiFieldConditionsInAction.add(fieldConditionConverter.toSDKFieldCondition());
+        }
+
+        com.silanis.esl.sdk.ReferencedFieldConditions sdkReferencedFieldConditions = new com.silanis.esl.sdk.ReferencedFieldConditions();
+        sdkReferencedFieldConditions.setReferencedInCondition(apiFieldConditionsInCondition);
+        sdkReferencedFieldConditions.setReferencedInAction(apiFieldConditionsInAction);
+        return sdkReferencedFieldConditions;
+    }
```

Comment:
When you create a new converter, you need to create a unit test for it.
Add ReferencedFieldConditionsConverterTest.

Reply:
done